backend/wmg/data/query.py

```diff
@@ -133,6 +133,16 @@ def cell_counts(self, criteria: WmgQueryCriteria, compare_dimension=None) -> Dat
         cell_counts.rename(columns={"n_cells": "n_total_cells"}, inplace=True)  # expressed & non-expressed cells
         return cell_counts
 
+    def cell_counts_df(self, criteria: WmgQueryCriteria) -> DataFrame:
```

Comment:
I added this query function to read the in-memory cell counts dataframe. This will eventually replace `cell_counts`